lib/web_ui/lib/src/engine/picture.dart

```diff
@@ -2,37 +2,34 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.6
+
 part of engine;
 
 /// An implementation of [ui.PictureRecorder] backed by a [RecordingCanvas].
 class EnginePictureRecorder implements ui.PictureRecorder {
   EnginePictureRecorder();
 
-  RecordingCanvas _canvas;
-  ui.Rect cullRect;
+  RecordingCanvas? _canvas;
+  ui.Rect? cullRect;
   bool _isRecording = false;
 
-  RecordingCanvas beginRecording(ui.Rect bounds) {
+  RecordingCanvas beginRecording(ui.Rect? bounds) {
```

Comment:
typically bounds is just cullRect in call sites, non-nullable?

Reply:
Done.